spec/features/course/assessment/answer/multiple_response_answer_spec.rb

```diff
@@ -54,6 +54,22 @@
           expect(all('.correct').length).to eq(options.to_a.count(&:correct?))
         end
       end
+
+      scenario 'I can view the auto grading results' do
+        answer = submission.answers.first
+        answer.auto_grade!
+        wait_for_grading(answer)
```

Comment:
I think you should trigger an auto grading then wait_for_job instead, since that is what the user would do.